frontend/src/components/App/Home/index.tsx

```diff
@@ -209,6 +209,23 @@ function useWarningSettingsPerCluster(clusterNames: string[]) {
   return warningLabels;
 }
 
+/**
+ * Gets the origin of a cluster.
+ *
+ * @param {Cluster} cluster
+ * @returns {string}
+ */
+function getOrigin(cluster: Cluster) {
+  if (cluster.meta_data?.source === 'kubeconfig') {
+    return `Kubeconfig (${process.env.KUBECONFIG ?? '~/.kube/config'})`;
```

Comment:
But maybe the translations could be added in this function, so they can be detected by that i18n script.

Reply:
added translations